packages/signed-api/src/handlers.ts

```diff
@@ -80,6 +82,14 @@ export const batchInsertData = async (
     return generateErrorResponse(400, message, detail ? { detail, signedData } : { signedData });
   }
 
+  if (env.LOG_APIDATA) {
+    // Log only the required fields to use less space, do not log the signature for security reasons.
+    const sanitizedData = batchSignedData.map((data) =>
+      pick(data, ['airnode', 'encodedValue', 'templateId', 'timestamp'])
+    );
+    logger.info('Received signed data.', { data: sanitizedData });
```

Comment:
nit: Maybe to emphasise that there was some logic done beforehand?

```suggestion
    logger.info('Received valid signed data.', { data: sanitizedData });
```